walk-in-interview/src/main/webapp/new-job.html

```diff
@@ -48,7 +48,7 @@
             <div class="job-field">
                 Job Duration (optional):
                 <select id="new-job-duration" name="job-duration">
-                    <option selected value="">Select</option>
+                    <option selected value="">Other</option>
```

Comment:
What do u mean by "Other"?

Reply:
I thought this might be more appropriate than the user leaving the job duration option as "select". Using "other" indicates a non specified job duration, which I think was our intention when we made job duration optional.